README.md

```diff
@@ -1,2 +1,183 @@
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Travelers Hub <a name="about-project"></a>
+
+
+**Space Travelers Hub** is a SPA web app that works with real live data from the SpaceX API, built for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://redux.js.org/">SpaceX API</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Reserve Rockets and Dragon]**
+- **[Join mission]**
+- **[Cancel Reservations]**
+- **[Leave Missions]**
+- **[Display Reserved Rockets and Dragons]**
+- **[Display Joined Mission]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Text Editor 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/danielochuba/space-travelers-hub.git
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd Space-travelers-Hub
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test 
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Daniel Ochuba**
+- GitHub: [@danielochuba](https://github.com/danielochuba)
+- Twitter: [@ochuba_daniel](https://twitter.com/ochuba_daniel)
+- LinkedIn: [Daniel Ochuba](www.linkedin.com/in/daniel-ochuba-ugochukwu)
+
+
+👤 **Sebastian Hernandez**
+
+- GitHub: [@shm04](https://github.com/shm04)
+- LinkedIn: [Sebastian Hernandez](https://www.linkedin.com/in/sebastian-hernandez-b42052259/)
+
+👤 **Ali Bello**
+
+- GitHub: [@devalibello](https://github.com/devalibello)
+- LinkedIn: [Ali Bello](https://www.linkedin.com/in/ali-bello-imoukhuede/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add more styles to Improve UI]**
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your Future Feature section more descriptive by adding more exciting features in this section 😊